terraform/ecs/main.tf

```
@@ -20,6 +20,15 @@ module "common" {
   aoc_version    = var.aoc_version
 }
 
+
+module "aoc_msk_cluster" {
+  source = "../data_aoc_msk"
+
+ cluster_version = var.kafka_version
+ testcase = var.testcase
+ dedup_topic = "ecs"
+}
```

This module is imported into every platform module.

Does it belong in `basic_components`? 
Will this module initialize for all test cases? Even non kafka ones? Should we be adding something like
```suggestion
module "aoc_msk_cluster" {
  # this syntax is wrong, don't copy it. It is merely a suggestion to get my point across. 
  count = string.startsWith(kafka, testcasename)
  source = "../data_aoc_msk"

 cluster_version = var.kafka_version
 testcase = var.testcase
 dedup_topic = "ecs"
}
```

---

This is is already done inside the `aoc_msk_cluster` module. If the cluster_version variable is empty, it will not fetch any data and it will return an empty map.